Makefile

```diff
@@ -22,7 +22,7 @@ help:
 	@awk 'BEGIN {FS = ":.*##"; printf "Usage: make ${FORMATTING_BEGIN_YELLOW}<target>${FORMATTING_END}\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  ${FORMATTING_BEGIN_BLUE}%-46s${FORMATTING_END} %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
 
-GOLANGCI_VERSION = 1.42.0
```

Comment:
Linter did not auto-fix due to dependencies issues. So I upgraded it. And it made me fix a couple of its complaints. It should be a separate PR, but I decided to try a shortcut.